controllers/admin/AdminTranslationsController.php

```diff
@@ -37,7 +37,7 @@ class AdminTranslationsControllerCore extends AdminController
     const CONTENT_TYPE_ACCEPTED = array('txt', 'tpl', 'html');
 
     /** @var string : Link which list all pack of language */
-    protected $link_lang_pack = 'http://www.prestashop.com/download/lang_packs/get_each_language_pack.php';
+    protected $link_lang_pack = 'http://i18n.prestashop.com/translations/'._PS_VERSION_.'/available_languages.json';
```

Comment:
Sorry, I can't.
`self::TRANSLATIONS_HOST.'/translations/%version%/%locale%/%locale%.zip'` won't pass in PHP 5.4